.github/workflows/test.yml

```diff
@@ -35,4 +35,17 @@ jobs:
       with:
         fetch-depth: 1
     - name: "Run unit tests"
-      run: go test -v .
\ No newline at end of file
+      run: go test -v .
+
+  test-unit-freebsd:
```

Comment:
is it possible to combine `test-unit` for multiple os using strategy matrix ?

Reply:
I'm not sure how to do that given that I need to use completely different steps. The only common thing is checkout.

Reply:
you can be inspired from this https://github.com/containerd/nerdctl/blob/main/.github/workflows/test.yml#L68-L96

Reply:
I've pushed updated version. To be honest, it looks absolutely disgusting :D

Reply:
Done.